src/styles/blocks/catalog.scss

```diff
@@ -18,6 +18,13 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition-duration: $duration;
```

Comment:
Please specify to which properties you want to apply the transition, because currently it applies to all the properties, and that's not good for performance.
https://developer.mozilla.org/en-US/docs/Web/CSS/transition

Reply:
I understand my mistakes. Both about new line and transition properties. Changed code according to the comments. Please check